src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -217,6 +219,24 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
       }
     }
   }
+  /* Return a Markdown table of all entries in the database */
```

Comment:
I could remove the invocation of it if you want, but I found it very helpful and informative as a developer (just sprinkling it in various places to make sure things are what I think they should be), so I left it in.  I was just frustrated by a lack of tooling to show me the symbol table, so I created this to help be able to visualize it better.